client/src/pages/Profile.js

```diff
@@ -0,0 +1,238 @@
+import React, { Component } from "react";
+
+import { Typography } from "@material-ui/core";
+import { withStyles } from "@material-ui/core/styles";
```

Comment:
looks like you aren't using this in this file right? 

Let's clean up all the warnings that create-react-app is probably throwing